users/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    # path('',views.user,name="user"),
+    path('register/', views.register, name='users-register'),
+    path('login/', views.login, name='users-register'),
```

Comment:
Can't use the same name for both the URL paths. Change the second one to `users-login`

Reply:
oh sorry 😓. it was copy-pasted from the line above